### PR TITLE
Remove Linky integration

### PR DESCRIPTION
**Pull request by Quentame**
## Proposed change

Linked to parent PR : home-assistant/core#38565

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core#38565
- Link to parent pull request in the Brands repository: home-assistant/brands#1775
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by Quentame** [comment]
As 'current' branch just got merged into the 'next' branch, I'm not sure if my choice is good.

**Review by frenck — CHANGES_REQUESTED** [review]
This is not cleaning up all link references across the site.
Please clean up all blog articles linking to this page, to prevent broken links on our website.

I've actually done it, but could not push onto this branch (because that was explicitly disabled by you).

**Comment by Quentame** [comment]
Cleaned all references.

First integration removal, so didn't know sorry.

My branch is on a fork, that's why.

**Comment by frenck** [comment]
@Quentame A fork is normally writable for members upstream, unless specifically disabled by the owner of the fork.

File: source/_posts/2018-10-01-release-79.markdown [hunk 1]
```diff
@@ -54,7 +54,6 @@ Oh, and yes, there is also some cool new integrations. Support has been added fo
 - Add Logi Circle component, camera and sensor platform ([@evanjd] - [#16540]) ([logi_circle docs]) ([camera.logi_circle docs]) ([sensor.logi_circle docs]) (new-platform)
 - deCONZ cover support ([@Kane610] - [#16759]) ([cover docs]) ([deconz docs]) ([cover.deconz docs]) (new-platform)
 - GeoJSON platform ([@exxamalte] - [#16610]) ([geo_location docs]) (new-platform)
-- Add linky sensor ([@tiste] - [#16468]) ([sensor.linky docs]) (new-platform)
```
Comment (frenck) [inline source/_posts/2018-10-01-release-79.markdown:57]: Don't remove those, as they are still valid. Just remove the links to the documentation.

Same applies to all other changes.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @Quentame 👍

**Pull request merged**